package.json

```
@@ -60,7 +60,7 @@
     "npm-run-all": "^4.1.5",
     "prettier": "^2.8.0",
     "remark-cli": "^11.0.0",
-    "stylelint": "^14.16.0"
+    "stylelint": "github:stylelint/stylelint#v15"
   },
   "peerDependencies": {
     "stylelint": "^14.14.0"
```

[suggest] It seems to update also the peer dep version:

```suggestion
    "stylelint": "^15.0.0"
```